test/JsonCollectorTest.php

```diff
@@ -154,22 +154,48 @@ public function testJsonErrors($sAdditionalDir, $sErrorMsg, $sExceptionMsg=false
             $this->oMockedLogger->expects($this->exactly(0))
                 ->method("Log");
         }
-        try{
-            $bResult = $oOrgCollector->Collect();
+	    try {
+		    $bResult = $oOrgCollector->Collect();
 
-            $this->assertEquals($sErrorMsg ? false : true, $bResult);
-        }
-        catch(Exception $e){
-             $this->assertEquals($sExceptionMsg, $e->getMessage());
-        }
+		    $this->assertEquals($sErrorMsg ? false : true, $bResult);
+	    } catch (Exception $e) {
+		    $this->assertEquals($sExceptionMsg, $e->getMessage());
+	    }
```

Comment:
```suggestion
			$this->assertEquals($sErrorMsg ? false : true, $bResult);
		} catch (Exception $e) {
			$this->assertEquals($sExceptionMsg, $e->getMessage());
		}
```

Reply:
Closing as this was solved globally
Thanks for your suggestion and efforts !